SocketRocket.podspec

```diff
@@ -11,8 +11,10 @@ Pod::Spec.new do |s|
 
   s.ios.deployment_target = '6.0'
   s.osx.deployment_target = '10.8'
+  s.tvos.deployment_target = '9.0'
```

Comment:
Apple's started versioning of tvOS from version 9.0 because it's a branch of iOS, I think.